tools/serverpod_cli/lib/src/util/string_manipulation.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:super_string/super_string.dart';
+
+List<String> splitIgnoringBrackets(
```

Comment:
Suggestion: Add documentation to this method with an example.

Since the thought here is to use it across the project I think it would be good to clearly explain it. Possibly even add separate unit tests to it. That way we ensure it functions as expected even if the usages of it are removed (that might have variations of it tested).